api/v1/database_types.go

```diff
@@ -40,105 +40,114 @@ const (
 	// +optional
 	Locale string `json:"locale,omitempty"`
 
-	// The LOCALE_PROVIDER (cannot be changed)
+	// Maps to the `LOCALE_PROVIDER` parameter of `CREATE DATABASE`. This setting cannot be changed.
 	// This option sets the locale provider for databases created in the new cluster.
 	// Available from PostgreSQL 16.
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="localeProvider is immutable"
 	// +optional
 	LocaleProvider string `json:"localeProvider,omitempty"`
 
-	// The LC_COLLATE (cannot be changed)
+	// Maps to the `LC_COLLATE` parameter of `CREATE DATABASE`. This setting cannot be changed.
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="localeCollate is immutable"
 	// +optional
 	LcCollate string `json:"localeCollate,omitempty"`
 
-	// The LC_CTYPE (cannot be changed)
+	// Maps to the `LC_CTYPE` parameter of `CREATE DATABASE`. This setting cannot be changed.
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="localeCType is immutable"
 	// +optional
 	LcCtype string `json:"localeCType,omitempty"`
 
-	// The ICU_LOCALE (cannot be changed)
+	// Maps to the `ICU_LOCALE` parameter of `CREATE DATABASE`. This setting cannot be changed.
 	// Specifies the ICU locale when the ICU provider is used.
 	// This option requires `localeProvider` to be set to `icu`.
 	// Available from PostgreSQL 15.
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="icuLocale is immutable"
 	// +optional
 	IcuLocale string `json:"icuLocale,omitempty"`
 
-	// The ICU_RULES (cannot be changed)
+	// Maps to the `ICU_RULES` parameter of `CREATE DATABASE`. This setting cannot be changed.
 	// Specifies additional collation rules to customize the behavior of the default collation.
 	// This option requires `localeProvider` to be set to `icu`.
 	// Available from PostgreSQL 16.
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="icuRules is immutable"
 	// +optional
 	IcuRules string `json:"icuRules,omitempty"`
 
-	// The BUILTIN_LOCALE (cannot be changed)
+	// Maps to the `BUILTIN_LOCALE` parameter of `CREATE DATABASE`. This setting cannot be changed.
 	// Specifies the locale name when the builtin provider is used.
 	// This option requires `localeProvider` to be set to `builtin`.
 	// Available from PostgreSQL 17.
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="builtinLocale is immutable"
 	// +optional
 	BuiltinLocale string `json:"builtinLocale,omitempty"`
 
-	// The COLLATION_VERSION (cannot be changed)
+	// Maps to the `COLLATION_VERSION` parameter of `CREATE DATABASE`. This setting cannot be changed.
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="collationVersion is immutable"
 	// +optional
 	CollationVersion string `json:"collationVersion,omitempty"`
 
-	// True when the database is a template
+	// Maps to the `IS_TEMPLATE` parameter of `CREATE DATABASE`.
```

Comment:
On this one and other re-settable parameters such as CONNECTION LIMIT.
I don't love that we say "Maps to ... parameter of `CREATE DATABASE`"
They also apply to ALTER.
What about just saying "Maps to  ... parameter for databases inPostgres"

Reply:
Good idea. Let me try.